version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "1.4.1-SNAPSHOT"
+version in ThisBuild := "3.0.0-SNAPSHOT"
```

Comment:
big bump but IMO, panda-hmac should be on the same version numbers as the other modules, but currently panda-hmac is on 2.2.0 and the rest of panda is on 1.4.0.
This could be 2.3.0 but it seems weird to me that most panda modules would not only jump to a new major version, but not even have a x.0.0 release? maybe i'm overthinking it though